web/config/settings/settings_base.py

```diff
@@ -258,6 +258,7 @@
 
 MAX_EXPORT_ATTEMPTS = 3
 FORCE_DOCX_FOOTNOTES = True
+FORCE_DOCX_SECTIONS = False
```

Comment:
Okay, one last question: if `FORCE_DOCX_SECTIONS = False`, then the results of the tests shouldn't change at all, right? 

I think every change to a test file in the repo is an instance of something slipping through.

These 28 byte shrinking of the complete casebook docx'es is probably just that thing I noticed about the "old" lua filter not being identical to the lua filter on develop.
![image](https://user-images.githubusercontent.com/11020492/152210961-342c8352-daa3-4b1d-a88e-5bf1cd009ac3.png)

The changes to the export of individual sections and resources might need some looking at.

Reply:
Got it. It was from one of those stupid, hilarious yet infuriating bugs that you cant see even though you're looking right at it— i was running a .replace on the template name like it had side effects rather than re-assigning the string value because I'm a professional.